globalwarmingpotentials.csv

```diff
@@ -1,98 +1,115 @@
-HFC134a,1300,1430,1300,1549,1530,4140,436,306
-HFC143,300,,328,397,364,1300,104,66.6
-HFC143a,3800,4470,4800,5508,5810,7840,1940,3250
-HFC152,,,16,20,21.5,77.6,6.14,3.89
-HFC152a,140,124,138,167,164,591,46.8,29.8
-HFC161,,,4,4,4.84,17.4,1.38,0.872
-HFC227ea,2900,3220,3350,3860,3600,5850,1100,1490
-HFC236cb,,,1210,1438,1350,3750,387,268
-HFC236ea,,,1330,1596,1500,4420,428,288
-HFC236fa,6300,9810,8060,8998,8690,7450,6040,8870
-HFC245ca,560,,716,863,787,2680,225,146
-HFC245fa,,1030,858,1032,962,3170,274,180
-HFC365mfc,,794,804,966,914,2920,261,172
-HFC4310mee,1300,1640,1650,1952,1600,3960,458,347
-SO2F2,,,4090,4732,4630,7510,1410,1920
-SF6,23900,22800,23500,26087,25200,18300,34100,30600
-NF3,,17200,16100,17885,17400,13400,18200,20000
-CF4,6500,7390,6630,7349,7380,5300,10600,9050
-C2F6,9200,12200,11100,12340,12400,8940,17500,15200
-C3F8,7000,8830,8900,9878,9290,6770,12400,11200
-cC4F8,8700,10300,9540,10592,10200,7400,13800,12400
-C4F10,7000,8860,9200,10213,10000,7300,13400,12100
-C5F12,7500,9160,8550,9484,9220,6680,12700,11200
-C6F14,7400,9300,7910,8780,8620,6260,11600,10500
-C7F16,,,7820,8681,8410,6120,11300,10200
-C8F18,,,7620,8456,8260,6010,11100,10000
-C10F18,,,7190,7977,7480,5480,9780,9010
-SF5CF3,,17700,17400,19396,18500,13900,21100,21600
-cC3F6,,,9200,10208,,,,
-HFE125,,14900,12400,13951,14300,13500,7680,13100
-HFE134,,6320,5560,6512,6630,12700,1940,2060
-HFE143a,,756,523,632,616,2170,176,113
-HCFE235da2,,350,491,595,539,1930,154,98.4
-HFE245cb2,,708,654,790,747,2630,213,137
-HFE245fa2,,659,812,981,878,3060,251,162
-HFE347mcc3,,575,530,641,576,2020,164,106
-HFE347pcf2,,580,889,1072,980,3370,279,181
-HFE356pcc3,,110,413,500,277,995,79,50.4
-HFE569sf2,,59,57,69,60.7,219,17.3,11
-HFE4310pccc124,,1870,2820,3353,3220,8720,920,647
-HFE236ca12,,2800,5350,6260,6060,11700,1770,1860
-HFE338pcc13,,1500,2910,3466,3320,9180,948,657
-HFE227ea,,,6450,7377,7520,9800,2570,4440
-HFE236ea2,,,1790,2143,2590,7020,741,521
-HFE236fa,,,979,1177,1100,3670,315,205
-HFE245fa1,,,828,997,934,3170,266,173
-HFE263fb2,,,1,2,,,,
-HFE329mcc2,,,3070,3598,3770,7550,1100,1090
-HFE338mcf2,,,929,1118,1040,3460,297,194
-HFE347mcf2,,,854,1028,963,3270,275,179
-HFE356mec3,,,387,468,264,949,75.3,48
-HFE356pcf2,,,719,867,831,2870,237,154
-HFE356pcf3,,,446,540,484,1730,138,88.4
-HFE365mcf3,,,,1,1.6,5.77,0.457,0.289
-HFE374pc2,,,627,758,12.5,45,3.56,2.25
-PFPMIE,,10300,9710,10789,10300,7750,11700,12000
-CHCl3,4,,16,20,20.6,74.2,5.87,3.72
-CH2Cl2,9,8.7,9,11,11.2,40.2,3.18,2.01
-CH3Cl,,13,12,15,5.54,19.9,1.58,1
-Halon1201,,,376,454,380,1340,108,69.8
+# Global warming potentials of greenhouse gases.,,,,,,,,,,,
+# Sources:,,,,,,,,,,,
+#   - SARGWP100: https://ghgprotocol.org/calculation-tools (specifically https://www.ghgprotocol.org/sites/default/files/ghgp/Global-Warming-Potential-Values%20%28Feb%2016%202016%29_1.pdf),,,,,,,,,,,
+#   - TARGWP100, TARGWP20, TARGWP500: https://www.ipcc.ch/site/assets/uploads/2018/03/WGI_TAR_full_report.pdf, page 388 Table 6.7,,,,,,,,,,,
```

Comment:
```suggestion
#   - TARGWP100, TARGWP20, TARGWP500: https://www.ipcc.ch/site/assets/uploads/2018/03/WGI_TAR_full_report.pdf, page 388 Table 6.7,,,,,,,,
```

To preserve GitHub's preview of the csv (this is stupid, but we appear to be trying to maintain this, see e.g. https://github.com/openclimatedata/globalwarmingpotentials/pull/27).

@rgieseke do we want to add an action to check that the number of rows is consistent?

Reply:
@znicholls Ye, that could be useful. Though, if one wants to keep the GitHub rendering used, actual readability might be improved by moving additional commas to the very end of the commented '#' lines and use ';' or something else to separate 'TARGWP100' and 'TARGWP20'.